src/ixdat/techniques/ms.py

```diff
@@ -414,8 +449,11 @@ def gas_flux_calibration_curve(
             tspan_bg (tspan): The time to use as a background
             ax (Axis): The axis on which to plot the ms_calibration curve result.
                 Defaults to a new axis.
-            axis_measurement (Axes): The MS plot axes to highlight the
-                ms_calibration on. Defaults to None. These axes are not returned.
+            axis_measurement (Axis): The MS plot axes to highlight the
+                ms_calibration on. Defaults to None.
+            tspans for ms_calibration is showing raw data or bg subtracted data
```

Comment:
I dont think this "tspans" is a defined argument in the function. So it can not be changed from the default "False".

Reply:
Seems like you missed or deleted some lines when copying the method.

Reply:
Addressed, answer in response to Anna's other comment.